OmiseSwift/API Models/Card.swift

```diff
@@ -204,6 +213,7 @@ public struct TokenizedCard: OmiseIdentifiableObject, OmiseLiveModeObject, Omise
     
     public let bankName: String?
     
+    public let firstDigits: FirstDigits
```

Comment:
I'm not sure if the first digits value will be available on every cards. I think it may be safer to have this as an optional.